akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -372,8 +372,13 @@ private[remote] object EndpointManager {
      */
     def markAsFailed(endpoint: ActorRef, timeOfRelease: Deadline): Unit =
       if (isWritable(endpoint)) {
-        addressToWritable += writableToAddress(endpoint) -> Gated(timeOfRelease)
-        writableToAddress -= endpoint
+        val address = writableToAddress(endpoint)
+        addressToWritable.get(address) match {
+          case Some(Quarantined(_, _)) ⇒ // don't overwrite Quarantined with Gated
+          case _ ⇒
```

Comment:
If I understand your words correctly, not that, AFAIK. 
if the previous policy was a Pass with Some refusedUid, must to overwrite to quarantined. If it was already quarantined or Gated, need to update timestamp.